src/raw/dockerapi/fetcher.go

```diff
@@ -35,24 +37,43 @@ func (f *Fetcher) Fetch(container types.ContainerJSON) (raw.Metrics, error) {
 	return metrics, nil
 }
 
-func (f *Fetcher) memoryMetrics(containerStats types.StatsJSON) raw.Memory {
-	panic("unimplemented")
+func (f *Fetcher) memoryMetrics(containerStats types.StatsJSON, hostConfig *container.HostConfig) raw.Memory {
+	mem := raw.Memory{}
+
+	if containerStats.MemoryStats.Usage != 0 {
+		mem.UsageLimit = containerStats.MemoryStats.Limit
+		mem.FuzzUsage = containerStats.MemoryStats.Usage
+	}
+
+	if hostConfig != nil {
+		// TODO: Disclaimer: we're inferring swap usage from provided metrics since Docker API doesn't expose it.
```

Comment:
Is the TODO set to check later if the value matches the one fetched when reading values from the filesystem?

Reply:
As talked, we're going to remove this function and set it to 0.